src/test/resources/application-test.yaml

```diff
@@ -15,6 +15,12 @@ keycloak:
 
 camunda-engine.servlet.context-path : /camunda
 
+shared-workflow-engine: ${SHARED_WORKFLOW_ENGINE:true}
```

Comment:
set the default value to true in code, also add same properties to tp-cloud-config and raise merge request